src/DataCollector/components/GroundTruthSelector.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Grid, TextField } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import _ from 'lodash';
+
+const GroundTruthSelector = ({ gt, setGroundTruth, items, index }) => {
+  return (
+    <Grid
+      container
+      spacing={2}
+      justify="flex-start"
+      alignItems="center"
+      style={{ minHeight: 90 }}
+    >
+      <Grid item style={{ padding: 0 }}>
+        <img
+          src={gt?.image}
+          alt={gt?.label}
+          style={{ minWidth: 80, width: 80 }}
+        />
+      </Grid>
+      <Grid item>
+        <Autocomplete
+          id="labels"
```

Comment:
I don't know if it is an error, but it looks like id will be not unique anymore.